b2share/modules/records/mappings/records/records.json

```diff
@@ -65,28 +65,41 @@
             "type" : "date",
             "format" : "strict_date_optional_time||epoch_millis"
           },
-          "alternate_identifier" : {
-            "type" : "string"
+          "alternate_identifiers" : {
+            "properties" : {
+              "alternate_identifier" : {
+                "type": "string"
+              }
+            }
           },
           "community" : {
             "type" : "string",
             "index": "not_analyzed"
           },
           "contact_email" : {
-            "type" : "string"
+            "type" : "string",
+            "index": "not_analyzed"
           },
           "creators" : {
-            "type" : "string"
+            "properties" : {
+              "creator" : {
+                "type": "string"
```

Comment:
`creator` is not of type `string` but of type `object`. It has a string subfield `creator_name`